### PR TITLE
feat(presets): add schedule presets for automerge

### PR DESCRIPTION
**Pull request by DuncanCasteleyn**
## Changes
I have made a copy of the schedule presets and modified the code so the same presets are available for `automergeSchedule`.

## Context
I think it would be useful to have the same schedule presets that exist for schedule, would also exist for `automergeSchedule`.

## Documentation

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work
I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

If I saw correctly presets are already tested and I ran this test, I also checked if the new md file for the automergeSchedule is generated for the docs.

### CODE REVIEW

**Review by rarkins — CHANGES_REQUESTED** [review]
Ideally we could keep it within the existing "schedule:" prefix

**Comment by DuncanCasteleyn** [comment]
> Ideally we could keep it within the existing "schedule:" prefix

Would you sugest they are prefixed with `:automerge`?

**Comment by DuncanCasteleyn** [comment]
Since none of the previous changes persist, force pushed.

**Review by rarkins — APPROVED** [review]

File: lib/config/presets/internal/schedule.ts [hunk 1]
```diff
@@ -2,46 +2,92 @@ import type { Preset } from '../types';
...
+const monthly = ['before 3am on the first day of the month'];
+const nonOfficeHours = [
+  'after 10pm every weekday',
+  'before 5am every weekday',
+  'every weekend',
+];
+const quarterly = ['every 3 months on the first day of the month'];
+const weekdays = ['every weekday'];
+const weekends = ['every weekend'];
+const yearly = ['every 12 months on the first day of the month'];
```
Comment (viceice) [inline lib/config/presets/internal/schedule.ts:16]: should we use cron syntax? 🤔

**Review by secustor — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**